Makefile

```diff
@@ -9,11 +9,17 @@ docs:
 	$(MAKE) -C docs
 	$(MAKE) -C assets
 
+pandoc:
+	$(MAKE) -C docs br-pandoc
+
+pandoc-deploy:
+	$(MAKE) -C docs br-pandoc-deploy
```

Comment:
I'm curious: Why the new target instead of replacing the existing target?

Reply:
Ah, see, this was me stabbing blindly at this without a fantastic understanding of what the previous set of Makefile invocations were doing. You can tell I don't write a lot of Makefiles. :) Honestly, we could swap out the whole Makefile most likely, since this just has to run three commands to generate the binary files and then a fourth to push them to S3.

Reply:
Maybe I’m old school, but I am actually fond of the makefile approach, especially for testing locally, although I realize with Docker that gets a little more complicated 😅

Happy to fix these up directly. I just wanted to make sure I wasn’t missing something :)

Reply:
...Except that we still have to generate the other documents using Weasyprint/Kramdown. Hmmm. This could be more of a challenge than anticipated, since the old Makefile builds all the .md files into .pdf.

Reply:
16f25f9 takes a blind stab at this, but I don't know how well it'll work. May have to step back and re-work this.

Reply:
Oh, right. OK, I can definitely roll with this some, so you don’t have to fight makefiles :) The docker setup alone is useful to identify the dependencies

Reply:
And while you didn't have to fight with Makefiles, I got to fight with Docker :)

The TL;DR: version of all of this:
* the 'pandoc' version available from trusty/xenial is too dang old
* the Docker images used lack make
* Unless we build our own custom docker image (which is an option, but annoying), we have to duplicate them in the `docker.sh`

I'm not thrilled with these options. The Makefile approach makes it easier to generate all of our artifacts from the relevant MD files, and I'd like to see us enable more. I don't think the `docker.sh` approach will scale well for us.

For now, though, it works, and can push for branches and trunk again. But only for the BRs.